sunpy/visualization/animator/base.py

```diff
@@ -540,8 +540,10 @@ def _sanitize_axis_ranges(self, axis_ranges, data_shape):
                 axis_ranges[i] = np.linspace(axis_ranges[i][0], axis_ranges[i][-1],
                                              data_shape[i]+1)
                 axis_ranges[i] = edges_to_centers_nd(axis_ranges[i], i)
-            elif len(axis_ranges[i]) == data_shape[i]+1:
+            elif axis_ranges[i].ndim == 1 and len(axis_ranges[i]) == data_shape[i]+1:
```

Comment:
Also, can you add a comment explaining why ```axis_ranges[i]``` may not be 1D?

Reply:
Okay, but the above part assumes that `axis_ranges[i]` is a `numpy` array, so it would be fair to expect one. However, I can make the changes if you think it is useful.

Reply:
Which above part?  I don't see where that's the case.  Not at least in a place from where the code can end up in this ```if``` statement.

Reply:
I meant [this](https://github.com/sunpy/sunpy/blob/master/sunpy/visualization/animator/base.py#L517) line.

Reply:
That code block only applies to ```image_axes```.  The code block here applies to ```slider_axes```.  So that line will never be executed for the given axis if the code makes into here.  So the check for being a numpy array needs to be done here as well.

Reply:
Or I can convert the `axis_ranges[i]` inside the loop to a `numpy` array, that should also suffice?